src/Form/Embeddable/IdentificationType.php

```diff
@@ -24,11 +24,14 @@ public function configureOptions(OptionsResolver $resolver): void
     {
         $resolver->setDefaults([
             'data_class' => Identification::class,
+            'is_required' => null,
```

Comment:
Should this be a boolean value?

Reply:
Yes.

Added allowed type check rather than setting value to `false` or `true`. This forces us to actively choose whether some `Identification` should be required or not.